R/transformCounts.R

```diff
@@ -97,6 +127,14 @@
 #'                       
 #' head(assay(tse, "clr"))
 #' 
+#' # Perform CSS normalization.
+#' tse <- transformAssay(tse, method = "css")
+#' head(assay(tse, "css"))
+#' 
+#' # Perform CSS fast variant normalization.
+#' tse <- transformAssay(tse, method = "css_fast")
```

Comment:
should be css.fast according to our naming conventions.

Reply:
ok thanks sorry haven't seen these changes till today, was travelling back to London yesterday, will make these changes and any other comments from the past few days this evening/tmrw morning if that's ok?

Reply:
yes, certainly - great if you can help to finalize!